tests/unittests/config/test_cc_growpart.py

```diff
@@ -136,10 +141,9 @@ def setUp(self):
         self.tmpdir = os.scandir("/tmp")
         self.tmpfile = open(self.tmppath, "w")
```

Comment:
We really should use pytest's tmpdir fixture if we can here to avoid using test constructs which write files directly to the top-level of the host system's `/tmp`  directory. In fact I'm not sure we need any of these operations in `setUp` anymore. They are antiquated and come from the python unittest framework and a misunderstanding of loggingsetup as well as temp file writing.